app/models/mixins/has_network_manager_mixin.rb

```diff
@@ -22,14 +20,15 @@ module HasManyCloudNetworksMixin
              :to        => :network_manager,
              :allow_nil => true
 
-    before_validation :ensure_managers
+    alias_method :all_cloud_networks, :cloud_networks
```

Comment:
see my note above. Imho this call in `provision_workflow` should be `cloud_networks`